components/autofill/Cargo.toml

```diff
@@ -19,8 +19,7 @@ serde_derive = "1"
 serde_json = "1"
 sql-support = { path = "../support/sql" }
 sync-guid = { path = "../support/guid", features = ["rusqlite_support", "random", "serde_support"] }
-sync15 = { path = "../sync15" }
-sync15-traits = {path = "../support/sync15-traits"}
+sync15 = { path = "../sync15", features = ["sync-engine"] }
```

Comment:
since the default is `sync-engine` do we need the `features` here? it's nice to be explicit so I don't mind it!

Reply:
Good question :) You are correct that we don't need it, but I too thought that explicit was better.